### DISCUSSION TITLE
Cannot flash SD card

### DISCUSSION THREAD

**Discussion by mbhforum** [tesla-android/issue-tracker#2] [Q&A] [answered]
Trying balenaEtcher or Rpi Etcher, both produced errors about unable to read the SD card when the flashing is complete. The operation still reports successful in the flashing app. When trying to boot a Rpi 4 with the SD Card, you get Boot Loader Image errors saying "Boot Image Header is Incorrect" I flashed the card with the Rpi OS and it worked fine, so it's not my MacOS desktop or the SD Card. I also tried a different MacOS and same issue happens there.  Anyone have any ideas on this?  

How do I access the console of the booted up RPi from a MacOS?  The RPi 4b is connected via USB-C (on my Rpi) to USB-A (on my Mac).

**Comment by mikegapinski** [comment]
After installing the deploy image use fastboot to install the main Android OS. It's all in the guide.

Good luck!

**Comment by mbhforum** [comment, marked as answer]
I got past this step.  It turns out the USB C -> A cable I was using to connect my RPi to my MacOS wasn't being used as a data cable, only power (so it was powering my Rpi up but not enough for my MacOS to recognize it).

**Comment by blu31c33** [comment]
did you end up using a USB hub or a different cable to resolve this issue?  i have tried tons of different cables and none of them recognize the pi when i run: ./fastboot devices.  i assume i'm having the same issue as you.  i tried 4 different usb a ports on the front of my mid-tower case plus a usb-a on the motherboard and a usb-c on the motherboard and cannot get anything to recognize the pi.  i assume all i have to do is out from usb-c on the pi.  

earlier i had a BUG error, that was because i had plugged a USB keyboard into the pi.  it DID NOT like that at all.  once i removed it, i got to the next step but cannot get past the issue with fastboot not seeing my pi.  i'm on windows 11.

**Comment by mbhforum** [comment]
I ended up using a different USB-C to USB-A cable (into my MacMini) and it worked.
